src/main/org/deidentifier/arx/distributed/ARXDistributedAnonymizer.java

```diff
@@ -304,6 +313,20 @@ public ARXDistributedResult anonymize(Data data,
         return new ARXDistributedResult(result, timePrepare, timeComplete, timeAnon, timeGlobalTransform, timePartitionByClass, timeSuppress, timeQuality, timePostprocess, qualityMetrics, maxMemory, numberOfMemoryMeasurements);
     }
 
+    private static int[] getSubset(ARXConfiguration config) {
+        int[] subset = null;
+        Set<PrivacyCriterion> privacyModels = config.getPrivacyModels();
+        for (PrivacyCriterion privacyCriterion : privacyModels) {
+            if (privacyCriterion.isSubsetAvailable()) {
+                subset = privacyCriterion.getDataSubset().getArray();
+            }
+        }
+        if (subset == null) {
+            return new int[0];
```

Comment:
This is very, very ugly. Why the hell are you doing that? Null is null and not int[0]. Please return null, if it is null and handle null as needed, unless there is a very good reason to do so (I dont think so, but feel free to try to convince me).